src/local_sensitivity/concrete_solve.jl

```diff
@@ -332,7 +332,13 @@ function DiffEqBase._concrete_solve_adjoint(prob,alg,sensealg::ReverseDiffAdjoin
   typeof(p) <: DiffEqBase.NullParameters || ReverseDiff.value!(tp, p)
   ReverseDiff.forward_pass!(tape)
   function reversediff_adjoint_backpass(ybar)
-    ReverseDiff.increment_deriv!(output, ybar)
+    if prob isa SDEProblem

```

Comment:
Is this a problem of SDEs only, i.e. does it work for ODEs? The type of `sol.u`, `sol.t` etc should usually be the same, I would assume.

Reply:
The problematic case was an SDE. I am assuming it worked for ODEs, since tests pass and all.

Reply:
It might be good to add the SDE example as a test.

Reply:
Yes

Reply:
But why does this happen? I I don't get it.